[2024-01-09T20:26:02.480+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: triggerdagallfailed.tsk2 manual__2024-01-09T20:26:01.162704+00:00 [queued]>
[2024-01-09T20:26:02.485+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: triggerdagallfailed.tsk2 manual__2024-01-09T20:26:01.162704+00:00 [queued]>
[2024-01-09T20:26:02.486+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-09T20:26:02.486+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-01-09T20:26:02.486+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-09T20:26:02.493+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk2> on 2024-01-09 20:26:01.162704+00:00
[2024-01-09T20:26:02.497+0000] {standard_task_runner.py:55} INFO - Started process 47390 to run task
[2024-01-09T20:26:02.502+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'triggerdagallfailed', 'tsk2', 'manual__2024-01-09T20:26:01.162704+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/7-trigger-dag.py', '--cfg-path', '/tmp/tmpgfooqb_0']
[2024-01-09T20:26:02.503+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask tsk2
[2024-01-09T20:26:02.557+0000] {task_command.py:388} INFO - Running <TaskInstance: triggerdagallfailed.tsk2 manual__2024-01-09T20:26:01.162704+00:00 [running]> on host 8c81727a976c
[2024-01-09T20:26:02.602+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=triggerdagallfailed
AIRFLOW_CTX_TASK_ID=tsk2
AIRFLOW_CTX_EXECUTION_DATE=2024-01-09T20:26:01.162704+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-09T20:26:01.162704+00:00
[2024-01-09T20:26:02.603+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-09T20:26:02.604+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2024-01-09T20:26:02.610+0000] {subprocess.py:86} INFO - Output:
[2024-01-09T20:26:07.620+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-09T20:26:07.646+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=triggerdagallfailed, task_id=tsk2, execution_date=20240109T202601, start_date=20240109T202602, end_date=20240109T202607
[2024-01-09T20:26:07.668+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-01-09T20:26:07.684+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
